tic_tac_toe/game.py

```diff
@@ -33,7 +50,12 @@ def _board_is_full(board):
 
     :param board: Game board.
     """
-    pass
+    for row in range(3):
+        for column in range(3):
+            if _position_is_empty_in_board((row, column), board):
+                return False
+
+    return True
```

Comment:
another option that passes the tests is
```
positions = [
    (0,0), (0,1), (0,2),
    (1,0), (1,1), (1,2),
    (2,0), (2,1), (2,2),
]

return '-' in positions
```

However, the loop in the first scenario breaks once it finds an already-occupied position on the board. Thus, it might be faster than this version. Must figure out how to benchmark both ways.